bin/traffic-splitter.js

```diff
@@ -7,29 +7,28 @@ const error = chalk.bold.red
 const logErr = (msg) => console.log(error(msg))
 
 program
-  .version('1.0.0')
+  .version(require('../package.json').version)
   .description('For better use of traffic-splitter create an isolated app. Take advantage of custom rules, handling events and adding middlewares.')
-  .option('-c, --conf <file>', 'provide configuration')
+  .option('-c, --conf <file>', 'provide configuration file (.json or .js with module.exports = {...})')
 
 program.on('/help', () => program.help())
 
 program.parse(process.argv)
 
 if (!program.conf) {
-  logErr('Configuration missing.')
-  logErr('Use: traffic-splitter -c yourConfiguration.json.')
+  logErr('Missing configuration file.')
+  logErr('Use: traffic-splitter -c yourConfigurationFile.')
   program.help()
 }
 
-if (!program.conf.endsWith('.json')) {
-  logErr('Configuration file must be a JSON file.')
-  process.exit(1)
-}
-
 const fs = require('fs')
 let config
 try {
-  config = JSON.parse(fs.readFileSync(program.conf))
+  if (program.conf.endsWith('.json')) {
+    config = JSON.parse(fs.readFileSync(program.conf))
+  } else {
+    config = require(process.cwd() + '/' + program.conf)
```

Comment:
You don't need to split your logic here, `require` knows how to load JSON files into a JavaScript object.

`config = require(process.cwd() + '/' + program.conf)` is all you need to load both js and json :)